### PR TITLE
firealpaca.com.FireAlpaca version 2.8.7.0

### PR DESCRIPTION
**Pull request by ghost**
- [ ] Have you signed the [Contributor License Agreement](https://cla.opensource.microsoft.com/microsoft/winget-pkgs)?
- [ ] Have you checked that there aren't other open [pull requests](https://github.com/microsoft/winget-pkgs/pulls) for the same manifest update/change?
- [ ] Have you [validated](https://github.com/microsoft/winget-pkgs/blob/master/AUTHORING_MANIFESTS.md#validation) your manifest locally with `winget validate --manifest <path>`? 
- [ ] Have you tested your manifest locally with `winget install --manifest <path>`?
- [ ] Does your manifest conform to the [1.2 schema](https://github.com/microsoft/winget-pkgs/tree/master/doc/manifest/schema/1.2.0)?

Note: `<path>` is the name of the directory containing the manifest you're submitting.

-----

###### Microsoft Reviewers: [Open in CodeFlow](https://portal.fabricbot.ms/api/codeflow?pullrequest=https://github.com/microsoft/winget-pkgs/pull/75603)

### CODE REVIEW

File: manifests/f/firealpaca/com/FireAlpaca/2.8.7.0/firealpaca.com.FireAlpaca.installer.yaml [hunk 1]
```diff
@@ -0,0 +1,13 @@
+# Created using wingetcreate 1.1.2.0
+# yaml-language-server: $schema=https://aka.ms/winget-manifest.installer.1.2.0.schema.json
+
+PackageIdentifier: firealpaca.com.FireAlpaca
```
Comment (Trenly) [inline manifests/f/firealpaca/com/FireAlpaca/2.8.7.0/firealpaca.com.FireAlpaca.installer.yaml:4]:
```suggestion
PackageIdentifier: FireAlpaca.FireAlpaca
```

File: manifests/f/firealpaca/com/FireAlpaca/2.8.7.0/firealpaca.com.FireAlpaca.locale.en-US.yaml [hunk 2]
```diff
@@ -0,0 +1,13 @@
+# Created using wingetcreate 1.1.2.0
+# yaml-language-server: $schema=https://aka.ms/winget-manifest.defaultLocale.1.2.0.schema.json
+
+PackageIdentifier: firealpaca.com.FireAlpaca
```
Comment (Trenly) [inline manifests/f/firealpaca/com/FireAlpaca/2.8.7.0/firealpaca.com.FireAlpaca.locale.en-US.yaml:4]:
```suggestion
PackageIdentifier: FireAlpaca.FireAlpaca
```

File: manifests/f/firealpaca/com/FireAlpaca/2.8.7.0/firealpaca.com.FireAlpaca.yaml [hunk 3]
```diff
@@ -0,0 +1,9 @@
+# Created using wingetcreate 1.1.2.0
+# yaml-language-server: $schema=https://aka.ms/winget-manifest.version.1.2.0.schema.json
+
+PackageIdentifier: firealpaca.com.FireAlpaca
```
Comment (Trenly) [inline manifests/f/firealpaca/com/FireAlpaca/2.8.7.0/firealpaca.com.FireAlpaca.yaml:4]:
```suggestion
PackageIdentifier: FireAlpaca.FireAlpaca
```

File: manifests/f/firealpaca/com/FireAlpaca/2.8.7.0/firealpaca.com.FireAlpaca.locale.en-US.yaml [hunk 4]
```diff
@@ -0,0 +1,13 @@
+# Created using wingetcreate 1.1.2.0
+# yaml-language-server: $schema=https://aka.ms/winget-manifest.defaultLocale.1.2.0.schema.json
+
+PackageIdentifier: firealpaca.com.FireAlpaca
+PackageVersion: 2.8.7.0
+PackageLocale: en-US
+Publisher: firealpaca.com
+PackageName: FireAlpaca
+License: freeware
+ShortDescription: FireAlpaca Setup
```
Comment (Trenly) [inline manifests/f/firealpaca/com/FireAlpaca/2.8.7.0/firealpaca.com.FireAlpaca.locale.en-US.yaml:10]: Please provide a better description of what FireAlpaca does

**Review by Trenly — CHANGES_REQUESTED** [review]

File: manifests/f/firealpaca/com/FireAlpaca/2.8.7.0/firealpaca.com.FireAlpaca.locale.en-US.yaml (hunk 4 above)
Comment (eliasjameshowell) [reply manifests/f/firealpaca/com/FireAlpaca/2.8.7.0/firealpaca.com.FireAlpaca.locale.en-US.yaml]: I've added a better description as requested.

**Pull request closed without merging**